workitem/workitemtype.go

```diff
@@ -180,6 +180,7 @@ func (wit WorkItemType) ConvertWorkItemStorageToModel(workItem WorkItemStorage)
 		if err != nil {
 			return nil, errors.WithStack(err)
 		}
+		result.Fields[SystemOrder] = workItem.ExecutionOrder
```

Comment:
@DhritiShikhar Maybe this was my fault. in the model representation of a work item, the execution order is stored on the root level and not in the fields. I assume, that is why you handle it separately.